components/SwipePage.jsx

```diff
@@ -1,76 +1,214 @@
+          color={colors.darkgrey}
+          size={30}
+          backgroundColor={colors.white}
+          borderWidth={1}
+          borderColor={colors.border}
+          onPress={() => handleSwipeBack()}
+        />
+        <Icon
+          name="closecircle"
+          size={70}
+          color={colors.red}
+          onPress={() => handleSwipeOnPress(-1)}
+        />
+        <Icon
+          name="checkcircle"
+          size={70}
+          color={colors.green}
+          onPress={() => handleSwipeOnPress(1)}
+        />
+        <IconButton
+          icon={(props) => <Icon name="heart" {...props} />}
+          color={colors.red}
+          size={30}
+          backgroundColor={colors.white}
+          borderWidth={1}
+          borderColor={colors.border}
+          onPress={() => handleAddToFavorite(clothesData[index])}
+        />
+      </View>
     );
+  };
+
+  const HeartIcon = () => {
+    return (
+      <View style={styles.heartContainer}>
+        <Icon name="heart" color={colors.red} style={{ width: 60 }} />
+        <Text>Added to Favorites</Text>
+      </View>
+    );
+  };
+
+  return (
+    <View style={styles.container}>
+      <Swiper
+        ref={swiperRef}
+        cards={clothesData}
+        cardIndex={index}
+        renderCard={(card) => <Card card={card} />}
+        onSwipedRight={() => handleSwipe(1)}
+        onSwipedLeft={() => handleSwipe(-1)}
+        onTapCard={(cardIndex) => {
+          const myTime = new Date();
+          const mySec = myTime.getTime();
+          if (mySec - lastTime < 250) {
+            handleAddToFavorite(clothesData[index]);
+            setTapCount(2);
+          }
+          setLastTime(mySec);
+        }}
+        stackSize={5}
+        stackSeparation={10}
+        infinite={true}
+        backgroundColor={colors.white}
+        verticalSwipe={false}
+        disableBottomSwipe
+        disableTopSwipe
+        style={styles.swiper}
+        animateCardOpacity
+        overlayLabels={{
+          left: {
+            title: "NOPE",
+            style: {
+              label: {
+                color: colors.white,
+                backgroundColor: colors.red,
+                padding: 15,
+                fontSize: 26,
+              },
+              wrapper: {
+                flexDirection: "column",
+                alignItems: "flex-end",
+                justifyContent: "flex-start",
+              },
+            },
+          },
+          right: {
+            title: "LIKE",
+            style: {
+              label: {
+                color: colors.white,
+                backgroundColor: colors.green,
+                padding: 15,
+                fontSize: 26,
+              },
+              wrapper: {
+                flexDirection: "column",
+                alignItems: "flex-start",
+                justifyContent: "flex-start",
+              },
+            },
+          },
+        }}
+      />
+      <Buttons />
+    </View>
+  );
 };
 
```

Comment:
Think we should decide between all styles using stylesheet or inline, not both together